injected/entry-points/extension-mv3.js

```diff
@@ -34,6 +35,16 @@ window.addEventListener(secret, ({ detail: encodedMessage }) => {
                 init(message.argumentsObject);
             }
             break;
+        default:
+            // Messages with messageType are subscription responses from the extension.
+            // Route them to the shared transport so all subscribed features receive them.
+            if (message.messageType) {
+                const transport = getSharedMessagingTransport();
```

Comment:
I think this all seems good, we might want to get a pass from @shakyShane on this and the other PR: https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/3319/files